### DISCUSSION TITLE
OctoDash without display?

### DISCUSSION THREAD

**Discussion by roboraptor** [UnchartedBull/OctoDash#1499] [Q&A] [answered]
Hello,

I know it sounds ridiculous but hear me out.

I use 7" tablet on the wall connected to OctoPrint WebUI.
From documentation I was not able to know if it works like some kind of app or is just "kiosked" webpage.
The second would be better because I could just redirect my tablet to other port/page/whatever.

So... Is this possible?

**Comment by UnchartedBull** [comment, marked as answer]
It is technically just a kiosked webpage, it does use some electron specific functionality (i.e. saving config and stuff). So it definitely isn't super easy to run this as a normal webpage, should be possible with some custom stuff. 

Another option would be to build OctoDash for your tablet (apk or ios app) and install that then. This is the builder OctoDash is using: https://www.electron.build/

**Reply by roboraptor** [reply]
It is 7" Lenovo with Android on it.

I am not really skilled at building something that big as an android app but I can definitely try :)

And is it already available on some port to view as a webpage? (Even without that special functionality)

Thank you for the informations :)

**Reply by UnchartedBull** [reply]
No, it's only available within the app. Should be pretty simple to build and android app though.
